### PR TITLE
ci(dh): API environment configuration

### LINKED ISSUE #136: Support deployment to multiple environments
## Acceptance criteria
- [x] Write backend-for-frontend base URL for the Development environment to JSON configuration which is deployed with the frontend application
- [ ] Coordinate CORS settings with backend developers
- [ ] Consider setting CSP settings
- [ ] Data flows from Metering point Testing environment to Backend-for-frontend Development environment to Frontend Development environment

### PR DESCRIPTION
**Pull request by LayZeeDK**
### Configuration
- Move `apiBase` setting to `assets/configuration/dh-api-environment.json`

#### Runtime and end-to-end tests
- Use `dh-api-environment.json` in Angular production mode
  - Will be added by a deployment workflow in #103 
  - ...or you can add it yourself to run the app locally against a deployed backend-for-frontend
- Fall back to `dh-api-environment.local.json` in Angular development mode

#### Unit and integration tests
- Use `dh-api-environment.local.json`

### Workspace boundaries
- Allow `environments` libraries to depend on `assets` libraries

## References
- Closes #136
- Blocked by #103

### CODE REVIEW

**Comment by dzhavat** [comment]
praise: Looks really nice! I like how the implementation picks the local environment file in development mode.

question: About the blocking tasks. I approve the PR but should we wait until the dependent tasks are done before merging? Should we add a "Blocked" label in that case?

**Review by dzhavat — APPROVED** [review]

**Comment by LayZeeDK** [comment]
It's already marked as blocked by #103 in ZenHub. Install the ZenHub web extension and you should be able to see it in GitHub.

**Pull request merged**
